http.go

```diff
@@ -216,6 +223,27 @@ func NewPricingClient(accountSid string, authToken string, httpClient *http.Clie
 	return c
 }
 
+func NewNotifyClient(accountSid string, authToken string, httpClient *http.Client) *Client {
+	if httpClient == nil {
```

Comment:
Any reason we can't use the `newNewClient` helper here? I understand the naming is poor...